setup.py

```diff
@@ -18,7 +18,7 @@
 from setuptools import setup, find_packages
 import sys
 
-install_requires = ['intelhex', 'six', 'enum34']
+install_requires = ['intelhex', 'six', 'enum34', 'future', 'websocket-client']
```

Comment:
websocket-client is very small library for websocket. I think that pyOCD used to have it in the past.